symphony-bdk-core/src/main/java/com/symphony/bdk/core/util/BdkExponentialFunction.java

```diff
@@ -0,0 +1,36 @@
+package com.symphony.bdk.core.util;
+
+import com.symphony.bdk.core.config.model.BdkRetryConfig;
+import io.github.resilience4j.core.IntervalFunction;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Custom BdkExponentialFunction class to be used in Bdk Retry mechanism.
```

Comment:
it is not explain the role of this class

Reply:
Okay I will explain it better